index.html

```diff
@@ -53,6 +54,246 @@ <h1>I'm Redwan.<br/>
+                        </div>
+                        <div class="tags">
+                            <ul>
+                                <li>
+                                    <a class="btn" href="">HTML</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">CSS</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">JavaScript</a>
+                                </li>
+                            </ul>
+                          </div>
+                          <div class="action">
+                            <button class="submit">See project</button>
+                        </div>
+                    </div>
+                </div>
+        </section>
+
+        <section class="section-bottom"> 
+            <div class="headerTop">
+                <h2>About Myself </h2>
+                <p >
+                    Hello I’m a software developer! I can 
+                    help you build a product , feature or 
+                    website. Look through some of my work 
+                    and experience! If you like what you see 
+                    and have a project you need coded, 
+                    don’t hesitate to contact me.
+                </p>
+            </div>
+            <!-- End of header top -->
+            <div class="headerBottom-bot">
+                <a href="#" class="link-text">LET’S CONNECT</a>
+                <div class="link-btn">
+                    <ul>
+                        <li>
+                            <a href="#"><img src="images/link-Vector4.svg" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Instagram2.png" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/link-Vector5.svg" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Twitter.svg" alt="Twitter"/></a>
+                        </li>
+                        
+                        <li>
+                            <a href="#"><img src="images/link-Vector3.svg" alt=""/></a>
+                        </li>
+                        
+                       
+                    </ul>
+                </div>
+                <div class="action">
+                    <button class="submit">Get my resume</button>
+                </div>
+                <div class="tags-bottom">
+                    <div class="container-links">
+                        <ul>
+                            <li class="lang"> 
+                                <div class="blink-name">Languages</div>
+                                <div class="blink-arrow"> <img class="Arrows" src="images/Down.png" alt=""></div> 
```

Comment:
- [ ] Great job adding the arrows to your design. Kinldy reduce their sizes to match design template.
### your design
![image](https://github.com/ethred/Portfolio/assets/94551484/3ce2434f-f02a-4a91-bb05-bc3095ee0e29)
### figma
![image](https://github.com/ethred/Portfolio/assets/94551484/3ec2b02e-e37e-4264-bbc6-a4b29bfc4da7)

Reply:
The initial reviewer highlighted the absence of arrows in the design.
Following the previous reviewer's feedback, the second reviewer requested a reduction in the size of the arrows to align with the Figma design.

- [ ] Let's resize the arrows to align with the Figma design (which we need to faithfully replicate).